apt-faster/action.yaml

```diff
@@ -0,0 +1,111 @@
+# Copyright 2025 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+name: "apt-faster"
+description: "Make apt go faster"
+
+runs:
+  using: composite
+  steps:
+    - name: Write apt-get wrapper
```

Comment:
I quite typing `apt-get` ages ago - should we also wrap `apt`?

Reply:
per [apt(8)](https://manpages.ubuntu.com/manpages/plucky/man8/apt.8.html):

> The apt(8) commandline is designed as an end-user tool and it may change behavior between versions. While it tries not  to break backward compatibility this is not guaranteed either if a change seems beneficial for interactive use.
>
> All features of apt(8) are available in dedicated APT tools like apt-get(8) and apt-cache(8) as well.  apt(8) just changes the default value of some options (see apt.conf(5) and specifically the Binary scope). So you should prefer using these commands (potentially with some additional options enabled) in your scripts as they keep backward compatibility as much as possible.

That is why i use `apt-get`

Reply:
this can be done follow up. i'm not opposed to wrapping that . i've never done it.